docs/get-started.md

```diff
@@ -107,160 +107,162 @@ Welcome to Nebula Graph (Version 0.1)
 nebula>
 ```
 
-Before query the dataset, you should switch to an existing graph space.
+If you have any questions or concerns about the deployment procedures, please do not hesitate to open an issue on git.
 
-```
-nebula> use nba
-Execution succeeded (Time spent: 154/793 us)
-```
-
----
+### Step 3: Build Your Own Graph
+This section describes how to build a graph and make queries. The example is built on the graph below:
 
-### Step 3 Simple Query Demo
+![Untitled Diagram (1)](https://user-images.githubusercontent.com/51590253/60649144-0774c980-9e74-11e9-86d6-bad1653e70ba.png)
 
-This query comes from a vertex walk up an edge.
-
-```
-nebula> GO FROM 5209979940224249985 OVER like
-========================
-|                   id |
-========================
-| -7187791973189815797 |
-------------------------
-|  3778194419743477824 |
-------------------------
-| -6952676908621237908 |
-------------------------
-........................
+There are three kinds of tags(course, building and team) and two edge types (select and like). The graph schema is:
+```json
+{
+     "tags": {
+       "course": ["name: string", "credits: integer"],
+       "building": ["name: string"],
+       "student": ["name: string", "age: integer", "gender: string"]
+     },
+     "edges": {
+        "select": ["grade: integer"],
+        "like": ["likeness: double"]
+     }
+}
 ```
 
-This query comes from a vertex walk up an edge and the target's age should be greater than or equal to 30, also renames the columns as `Player`, `Friend` and `Age`.
+#### Create a Graph Space
+<em>Space</em> is a region that provides physically isolation of graphs in Nebula. First we need to create a space and use it before other operations.
```

Comment:
Try not to use HTML tags in Markdown files, since they are not supported officially and consistently.